### ISSUE TITLE
byteData_ is empty object

### ISSUE THREAD

**Issue by diwu86** [rochars/wavefile#6]
Hi I use this in my react native project. When I try to load wav file from data no matter base64 or a buffer, I always got error that byteData_ is an empty so things like byteData_.types is undefined. I had to change all other byteData_ to byteData in [file](https://github.com/rochars/wavefile/blob/master/index.js) except the definition one [here](https://github.com/rochars/wavefile/blob/master/index.js#L36). I also had to do the same thing for riffChunks_. Can someone help me with this? I don't quite get why byteData without the underscore has all the information. Thanks

**Comment by rochars** [comment]
What version of WaveFile are you using? Also, could you give me more information about how WaveFile is being used in your React Native app?

Cheers!
Rafael

**Comment by diwu86** [comment]
wavefile": "^6.13.0
```
import WaveFile from 'wavefile'
import RNFS from 'react-native-fs'

RNFS.readFile(path, 'base64').then(data => {
let wav = new WaveFile()
wav.fromBase64(data)
}
```
the path is the wav file path.

I try to load data to buffer and get the same error

**Comment by rochars** [comment]
Could you try installing wavefile@7.0.0-alpha.8 and see if this is still an issue on your build?

Cheers!
Rafael

**Comment by diwu86** [comment]
Now when I do ```let wav = new WaveFile()```, I got ```TypeError: _wavefile2.default is not a constructor```

**Comment by rochars** [comment]
In version 6:
WaveFile = require('wavefile');

In version 7:
WaveFile = require('../dist/wavefile.umd.js').WaveFile;

Sorry not to mention that earlier. Did this help?

**Comment by diwu86** [comment]
Yes it's working now. Thank you very much! Are you going to release 7 with the fix?

**Comment by rochars** [comment]
:D 

Yes, but there are more things to do before the release. I'm actually working on byte-data and riff-chunks (dependencies of WaveFile) to fix memory issues. Version 7 will be out after that.

Cheers!
Rafael

**Comment by rochars** [comment]
Fixed in 7.0.0
